ovmf-vars-generator

```diff
@@ -51,18 +63,14 @@ def generate_qemu_cmd(args, readonly, *extra_args):
         '-no-user-config',
         '-nodefaults',
         '-m', '768',
-        '-smp', '2,sockets=2,cores=1,threads=1',
-        '-chardev', 'pty,id=charserial1',
-        '-device', 'isa-serial,chardev=charserial1,id=serial1',
-        '-global', 'driver=cfi.pflash01,property=secure,value=%s' % (
-            'off' if args.disable_smm else 'on'),
+        '-smp', '1,sockets=1,cores=1,threads=1',
```

Comment:
Why have less CPUs on ARM?

Reply:
I can't remember exactly why I reduced that, but IIRC it was because initially I was testing this in a VM with only a single core. 

Frankly, a better question is why have more than 1 core, the entire firmware stack is single threaded.

Reply:
> Frankly, a better question is why have more than 1 core, the entire firmware stack is single threaded.

OVMF includes modules for providing EFI_PEI_MP_SERVICES_PPI, EFI_MP_SERVICES_PROTOCOL, and EFI_MM_MP_PROTOCOL. So the general statement that the entire firmware stack is single threaded is incorrect.

It is true that PI / UEFI services are generally only usable on the BSP.

It is also true that, for this particular use case, we do not need more than 1 VCPU.